### ISSUE TITLE
[Bug]: NOVA don`t play *.m3u8

### ISSUE THREAD

**Issue by ansilent** [nova-video-player/aos-AVP#1096]
### Problem description

NOVA don`t play *.m3u8

### Steps to reproduce the issue

When I open a video in the Lampa TV app I see a black screen. The link to the video has the format https://хх.ххххх.ххх/content/stream/new/ххххххххх.хххх.1080p.webdl.ukr.line_115505/hls/1080/index.m3u8

Inside the index.m3u8 :

#EXTM3U
#EXT-X-VERSION:3
#EXT-X-ALLOW-CACHE:YES
#EXT-X-TARGETDURATION:10
#EXT-X-MEDIA-SEQUENCE:1
#EXT-X-PLAYLIST-TYPE:VOD
#EXTINF:5.005000,
segment1.ts
#EXTINF:5.880878,
segment2.ts
#EXTINF:6.423078,
segment3.ts
#EXTINF:5.005000,
segment4.ts
#EXTINF:5.630633,
segment5.ts
#EXTINF:5.005000,
segment6.ts
#EXTINF:5.005000,
segment7.ts
#EXTINF:5.005000,
segment8.ts
.
.
.
.
.
.
.
.
.
.
.
.
.
.
#EXT-X-ENDLIST

### Your phone/tablet/androidTV model

Mecool KM2

### Operating system version

Android TV 12

**Comment by phhusson** [comment]
What you want is not m3u8 support, it's HLS support. It normally doesn't make sense for a video player to read HLS streams, because it's usually streams, not "file video".
Maybe your precise use-case makes sense, and we could look at it, but you're giving too little information as to your usecase for us to decide.

**Comment by ansilent** [comment]
Thanks for your reply.
What additional information should I provide?

**Comment by phhusson** [comment]
Explain what is your usecase.

**Comment by ansilent** [comment]
I use Lampa TV app to watch movies. The Lampa TV is a free application for watching movies online. This application parses links to movies from various Internet resources. The links will open in third-party media players. And I want to use Nova to watch selected movie.
For example parsed link has the format https://хх.ххххх.ххх/content/stream/new/ххххххххх.хххх.1080p.webdl.ukr.line_115505/hls/1080/index.m3u8

**Comment by phhusson** [comment]
That's https://lampa.mx right?

**Comment by ansilent** [comment]
Yes. Client browser for Android https://github.com/lampa-app/LAMPA

**Comment by phhusson** [comment]
Understood, thanks.
So, I'm sorry, but our primary target is to handle a collection of video files, not play back every internet stream. Since you're not using Nova Video Player to handle your collection, we feel that your use-case doesn't fit what we intent to do.
For a "simple" video player that supports HLS, you can find many others. For your use case, I recommend Just Player which should handle your HLS just fine.
